team-members/valerii-tanachov.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="../styles/valerii-tanachov.css">
+    <title>Resume</title>
+  </head>
+  <body>
+    <!-- Header -->
```

Comment:
you can put this part into header tag in order for search engines to understand it better)